### DISCUSSION TITLE
Welcome to the Physion Discussions tab!

### DISCUSSION THREAD

**Discussion by nikoniko78** [nikoniko78/scripts-java-physion#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Carstyle476** [comment]
Hello! I'm Carstyle, on the site my name is Aiden. I do funny stuff with physics. B)

**Reply by nikoniko78** [reply]
Hello Carstyle! Thanks for joining! I currently have a few scripts under your credit, so you can change them to whatever you please, and can add your own. 

A badge for you:
![Screenshot 2024-11-19 122356](https://github.com/user-attachments/assets/8ed3f0da-5b70-4ff9-ae7e-104d778da571)
